I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 386, in script
    call screen look_around
  File "renpy/common/000statements.rpy", line 519, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/script.rpy", line 359, in execute
    screen look_around():
  File "game/script.rpy", line 359, in execute
    screen look_around():
  File "game/script.rpy", line 367, in execute
    imagebutton:
  File "game/script.rpy", line 367, in keywords
    imagebutton:
NameError: name 'Displayable' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 386, in script
    call screen look_around
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 1859, in execute
    self.call("execute")
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 1847, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\statements.py", line 179, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 519, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\exports.py", line 2724, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 2662, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 3049, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 521, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 521, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 521, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\screen.py", line 424, in visit_all
    callback(self)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\core.py", line 3049, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\screen.py", line 434, in per_interact
    self.update()
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\display\screen.py", line 619, in update
    self.screen.function(**self.scope)
  File "game/script.rpy", line 359, in execute
    screen look_around():
  File "game/script.rpy", line 359, in execute
    screen look_around():
  File "game/script.rpy", line 367, in execute
    imagebutton:
  File "game/script.rpy", line 367, in keywords
    imagebutton:
  File "<screen language>", line 370, in <module>
NameError: name 'Displayable' is not defined

Windows-8-6.2.9200
Ren'Py 7.1.1.929
SBU Senior Project 1.0
Tue Jan 01 16:48:19 2019
